packages/prisma/.env.sample

```diff
@@ -1 +1,2 @@
 DATABASE_URL="postgresql://username:password@db:5432/postgres?schema=public&connect_timeout=300"
+DATABASE_URL="postgresql://postgres:password@localhost:5432/postgres?schema=public&connect_timeout=300"
```

Comment:
`postgres` is the default username so we prefer to keep that in our sample env files